public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/RH-logo-no-text.ico" />
```

Comment:
favicon is special icon and it should be called like that. It is only used for tab icon. So rename it to favicon